### ISSUE TITLE
Define the URL routes and views for the Changelogs app

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/experimenter#8394]
* Define the URL routes for accessing the app's views
* Create the views for rendering the app's templates
* Add any necessary context to the template rendering process

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3224)

**Comment by KramStyles** [comment]
Hello @yashikakhurana I'm an OutReachy participant and I'd love to work on this issue. Can I work on it?

**Comment by yashikakhurana** [comment]
Hi @KramStyles thank you for your interest, this task is blocked once we land this - https://github.com/mozilla/experimenter/issues/8393 , if you want I can assign you and you can work on this once it is landed. I will adding more tickets related to this app too :) , let me know if you want to get this ticket assigned

**Comment by KramStyles** [comment]
Yes, I'd love to work on this once it's landed. I'd want to get assigned to this ticket and start working on it once it's unblocked. Thank you very much

**Comment by skynette** [comment]
Is it possible for me to also work on this along side @KramStyles ?

**Comment by yashikakhurana** [comment]
@skynette I won't mind but it would be better if you do another issue as it will be used in the evaluation process for selecting candidate, I added another task which could be handled once this issue get landed- https://github.com/mozilla/experimenter/issues/8414

**Comment by skynette** [comment]
okay thanks

**Comment by yashikakhurana** [comment]
@skynette are you interested in taking that issue? if yes I will assign it to you as you first asked, otherwise there are 2 more applicants on queue waiting to get this issue assigned, please let me know

**Comment by skynette** [comment]
Yes I am interested in taking the issue

**Comment by yashikakhurana** [comment]
@skynette do you mind taking another issue, I already assigned it to @KramStyles , let me know if you want any help

**Comment by skynette** [comment]
okay I will find another

**Comment by yashikakhurana** [comment]
@KramStyles  I am so sorry, I have to remove this issue for Outreachy tasks, please feel free to look at other issues, and let me know if you need any help.

**Comment by KramStyles** [comment]
Hello @yashikakhurana can I still work on a task if it's not tagged OutReachy? If it's something I can do, can I still attempt it?

**Comment by yashikakhurana** [comment]
@KramStyles unfortunately no, thanks for asking
